packages/falter-common/files/etc/uci-defaults/freifunk-berlin-dhcp-defaults

```diff
@@ -31,4 +31,8 @@ uci set dhcp.frei_funk_ipv6.ip="$router_ula"
 # to us, that we can't forward now
 uci add_list dhcp.lan.dhcp_option='3'
 
+# default DNS, will be overwritten by community profile later on
+uci add_list dhcp.@dnsmasq[0].server='5.9.164.112' # dns3.digitalcourage.de
+uci add_list dhcp.@dnsmasq[0].server='80.67.169.40' # ns1.fdn.fr
+
```

Comment:
Ich denke, wir könnten hier ruhig auch die 9.9.9.10 setzen (Quad9 ganz ohne Filter).

Reply:
Gerne in einem baldigen separaten PR :) Hab das aus dem Wizard übernommen